### PR TITLE
	RavenDB-7809 Create an web socket endpoint to get the outgoing

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Server/Documents/Handlers/ReplicationHandler.cs [hunk 1]
```diff
@@ -236,10 +235,64 @@ public Task Performance()
             return true;
         }
 
+        [RavenAction("/databases/*/replication/pulses/live", "GET", AuthorizationStatus.ValidUser, SkipUsagesCount = true)]
+        public async Task PulsesLive()
```
Comment (ml054) [inline src/Raven.Server/Documents/Handlers/ReplicationHandler.cs:239]: I won't call that endpoint replication pulses, as we want to have single web socket for pulses for:
replication (internal & external), sql etl, raven db etl

Comment (fitzchak) [reply src/Raven.Server/Documents/Handlers/ReplicationHandler.cs:239]: ok, I'll change this and will add more pulses.

Comment (ppekrol) [reply src/Raven.Server/Documents/Handlers/ReplicationHandler.cs:239]: maybe stick to one word? 'Heartbeat'? It is a bit confusing that we are naming same thing using two different words.

Comment (ml054) [reply src/Raven.Server/Documents/Handlers/ReplicationHandler.cs:239]: Based on types:
https://github.com/ravendb/ravendb/pull/3151/files#diff-7ba6900cefee273c07cc76b10b9c7265R93

I think naming it hearbeat is not correct, as we need some broader term. Maybe: 'event' , or taskEvent?

Comment (ayende) [reply src/Raven.Server/Documents/Handlers/ReplicationHandler.cs:239]: `event` and `taskEvent` are so generic we might as well call them `item1` 
Let us cal this `replication/live-tracking` or something like this?

**Pull request merged**